diff -urN 1.org/f2c.h0 1/f2c.h0
--- 1.org/f2c.h0	1999-09-07 19:46:24.000000000 +0200
+++ 1/f2c.h0	2012-07-29 14:10:11.654966145 +0200
@@ -154,16 +154,6 @@
 	};
 typedef struct Namelist Namelist;
 
-#define abs(x) ((x) >= 0 ? (x) : -(x))
-#define dabs(x) (doublereal)abs(x)
-#define min(a,b) ((a) <= (b) ? (a) : (b))
-#define max(a,b) ((a) >= (b) ? (a) : (b))
-#define dmin(a,b) (doublereal)min(a,b)
-#define dmax(a,b) (doublereal)max(a,b)
-#define bit_test(a,b)	((a) >> (b) & 1)
-#define bit_clear(a,b)	((a) & ~((uinteger)1 << (b)))
-#define bit_set(a,b)	((a) |  ((uinteger)1 << (b)))
-
 /* procedure parameter types for -A and -C++ */
 
 #define F2C_proc_par_types 1
